README.md

```diff
@@ -8,32 +8,34 @@
 
 [![TypeDB Studio](./images/studio_full_1.png)](./images/studio_full_1.png)
 
-TypeDB Studio is TypeDB's Integrated Development Environment to perform knowledge engineering.
+TypeDB Studio is a fully-featured Integrated Development Environment (IDE) for TypeDB.
 
-You can utilise Studio like [TypeDB Console](https://docs.vaticle.com/docs/console/console)
-and [TypeDB Clients](https://docs.vaticle.com/docs/client-api/overview), to connect to your TypeDB instance and perform
+You can utilise Studio as you would [TypeDB Console](https://docs.vaticle.com/docs/console/console)
+and the [TypeDB Clients](https://docs.vaticle.com/docs/client-api/overview): to connect to your TypeDB instance and perform
 queries.
 
 Studio is an IDE designed for the end-to-end development of TypeDB databases, database queries, and data exploration,
-via a clean and intuitive UI.
+via a clean and intuitive UI that gives you a top-down view of your entire database.
 
 ## Usage
 
-Upon launching Studio, you'll be asked for your TypeDB server details and to choose or create a database. Studio makes
-these choices explicit so that it's always clear what you're working on at any given time.
```

Comment:
The flow is clearer from 'Usage' into its sub-headings now, and we've removed a philosophical sentence about explicit choices that isn't relevant.